pkg/install/v1alpha1/jiva_volume_0.7.0.go

```diff
@@ -217,6 +217,7 @@ spec:
     - jiva-volume-delete-listtargetservice-default-0.7.0
     - jiva-volume-delete-listtargetdeployment-default-0.7.0
     - jiva-volume-delete-listreplicadeployment-default-0.7.0
+    - jiva-volume-delete-deletedata-default-0.7.0
```

Comment:
@AmitKumarDas -- wondering if we can revert the changes to this file. We could probably raise another PR with updates to CAS templates on how to delete. This way the dependency on this PR with the availability of delete api can be removed and this PR can be merged.